### ISSUE TITLE
winetricks -q msxml3 stopped working on wine devel 4.8 (4.7 ok) 32 & 64 Bit

### ISSUE THREAD

**Issue by bitranox** [Winetricks/winetricks#1230]
winetricks -q msxml3 stopped working on wine devel 4.8 32/64 Bit
it worked on wine devel 4.7 but stopped working on wine devel 4.8 !

works : wine stable wine-4.0 32 Bit / 64 Bit
works : wine devel wine-4.7 32 Bit / 64 Bit
works : wine staging wine-4.7 32 Bit / 64 Bit

FAILS : wine devel wine-4.8 32 Bit / 64 Bit

install procedure used : 
```
sudo apt-get install -y cabextract
apt-get install -y libxml2
apt-get install -y libpng-dev
wget https://dl.winehq.org/wine-builds/winehq.key
sudo apt-key add winehq.key
sudo apt-add-repository 'deb https://dl.winehq.org/wine-builds/ubuntu/ xenial main'
sudo apt-get update
sudo apt-get install --install-recommends winehq-devel
sudo wget --directory-prefix=/usr/bin/ https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
sudo chmod +x /usr/bin/winetricks
sudo winetricks -q --self-update
winetricks nocrashdialog
winetricks -q windowscodecs   # passes 
winetricks -q msxml3   # hangs

```

I make daily builds on travis for my WINE environment - see : 

https://travis-ci.org/bitranox/install_python_on_wine_and_travis

**Comment by austin987** [comment]
That would likely be a wine regression. If possible, run a regression test and report and file upstream. All I can easily do is add a warning; finding a workaround likely isn't worth it since this probably be fixed quickly upstream.

**Comment by bitranox** [comment]
I filed a bug report at WineHQ, thanks for Your answer

**Comment by austin987** [comment]
Thanks. Looks like https://bugs.winehq.org/show_bug.cgi?id=47185 already has a patch, https://source.winehq.org/patches/data/164554, so I expect this will be fixed in 4.9.

**Comment by austin987** [comment]
Fixed upstream.
